docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -132,26 +132,135 @@ __Example__:
     platformName: "Windows 10"
 ```
 
-### `testMatch`
+### `screenResolution`
 
-__Description__: The explicit name, regex, or location of the test files.
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
 
-__Type__: *string* | *regex*
+__Description__: Set browser window screen resolution.
+
+__Type__: *string*
 
 __Example__:
 ```yaml
-    testMatch: '**/*.js'
+    screenResolution: "1920x1080"
 ```
 
-### `screenResolution`
+### `disableScreenshots`
+__Description__: Prevents TestCafe from taking screenshots. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#disablescreenshots).
 
-<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
+__Type__: *boolean*
 
-__Description__: Set browser window screen resolution.
+__Example__:
+```yaml
+  disableScreenshots: true
+```
+
+### `tsConfigPath`
+__Description__: The absolute or relative path to the TypeScript configuration file. Relative paths are resolved against the current directory (the directory from which you run TestCafe).
 
 __Type__: *string*
 
 __Example__:
 ```yaml
-    screenResolution: "1920x1080"
+  tsConfigPath: /path/to/file
+```
+
+### `clientScripts`
+__Description__: Injects scripts into all pages visited during the test. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/test-api/fixture/clientscripts.html).
+
+__Type__: *object*
```

Comment:
Since you can provide multiple values, would this be an array?

Reply:
yep. You're right. updating.